Source/Krypton Components/Krypton.Toolkit/Utilities/ImageNativeMethods.cs

```diff
@@ -1,8 +1,14 @@
-﻿// *****************************************************************************
-// BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit)
-// by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2022. All rights reserved. 
-//  Version 6.0.0  
-// *****************************************************************************
+﻿#region BSD License
```

Comment:
These are new files and have nothing to do with the original, hence not included

Reply:
Done

Reply:
Reverted